src/styles/muiTheme.ts

```diff
@@ -57,6 +57,7 @@ export const theme = createTheme({
     MuiPopper: {
       styleOverrides: {
         root: {
+          zIndex: '1800 !important',
```

Comment:
Are you sure that it will not cause the same issue as we encountered yesterday? 👀

Reply:
Good catch, it would! I removed it in another PR and rebase bru he's it back, will remove